### DISCUSSION TITLE
Regarding error because of Java heap Space

### DISCUSSION THREAD

**Discussion by pmanandhar** [atoti/atoti#600] [Q&A]
How do I deal with the issue of java heap space? 

I am getting the following exception:
AtotiJavaException: Java heap space

What is the efficient way to deal with this issue and what is the recommended size of heap size as compared to RAM size?

Thanks

**Comment by CamilleBlanco** [comment]
Hi,
You can find general advice on memory usage in [this discussion](https://github.com/atoti/atoti/discussions/525).
Let me know if it helps 😄
